### ISSUE TITLE
BMC goes unstable on resets

### ISSUE THREAD

**Issue by gkeishin** [openbmc/openbmc#3379]
This is intermittent but has been seen lately in master build few times.
```
root@witherspoon:~# obmcutil state
CurrentBMCState     : xyz.openbmc_project.State.BMC.BMCState.NotReady
CurrentPowerState   : xyz.openbmc_project.State.Chassis.PowerState.Off
DBus error occurred: The name xyz.openbmc_project.State.Host was not provided by any .service files
```
```
root@witherspoon:~# obmcutil power
pgood = 0
state = 0
pgood_timeout = 10
root@witherspoon:~# uptime
 13:25:50 up  3:12,  load average: 0.83, 1.13, 1.40
root@witherspoon:~# uname -a
Linux witherspoon 4.18.7-b7ba2a0af62b835fb3cac7bd3bbc72dcb1955b65 #1 Wed Sep 12 19:22:06 UTC 2018 armv6l GNU/Linux
```
```
root@witherspoon:~# cat /etc/os-release
ID="openbmc-phosphor"
NAME="Phosphor OpenBMC (Phosphor OpenBMC Project Reference Distro)"
VERSION="v2.3-332"
VERSION_ID="v2.3-332-g5f540a2"
PRETTY_NAME="Phosphor OpenBMC (Phosphor OpenBMC Project Reference Distro) v2.3-332"
BUILD_ID="v2.3"
root@witherspoon:~#
```
`Process top snapshot`
```
Mem: 265968K used, 163552K free, 1308K shrd, 13476K buff, 50072K cached
CPU:   0% usr 100% sys   0% nic   0% idle   0% io   0% irq   0% sirq
Load average: 19.99 10.67 4.39 17/137 2179
  PID  PPID USER     STAT   VSZ %VSZ %CPU COMMAND
 1696     1 root     RN   99996  23%  43% /lib/systemd/systemd-coredump
  558     2 root     RW       0   0%  35% [ubi_bgt0d]
 2176  1108 root     R     3064   1%   9% top -n 1 -b
  734     1 root     S    22892   5%   7% /lib/systemd/systemd-journald
    1     0 root     R    35444   8%   2% /lib/systemd/systemd
 1853     1 root     S     6484   2%   2% phosphor-dump-manager
 1388     1 root     RN    4748   1%   2% /lib/systemd/systemd-coredump
 1328     1 root     RN   99996  23%   0% /lib/systemd/systemd-coredump
 1556     1 root     RN   99996  23%   0% /lib/systemd/systemd-coredump
 1392     1 root     DN   99996  23%   0% /lib/systemd/systemd-coredump
 1360     1 root     RN   99996  23%   0% /lib/systemd/systemd-coredump
 1698     1 root     RN   99996  23%   0% /lib/systemd/systemd-coredump
 2179     1 root     R    35444   8%   0% /lib/systemd/systemd
 1906     1 root     S    33808   8%   0% power_control.exe 500 10
 1903     1 root     S    33804   8%   0% button_reset.exe
 1904     1 root     S    33804   8%   0% button_power.exe
 1428     1 root     S    31600   7%   0% python /usr/sbin/phosphor-gevent rest_dbus --no-ssl
 1039     1 root     S    19424   5%   0% /usr/sbin/rsyslogd -n
 1134  1129 www-data S    15640   4%   0% nginx: worker process
 1129     1 root     S    15500   4%   0% nginx: master process nginx -p /var/volatile/nginx
 1012     1 systemd- S    15252   4%   0% /lib/systemd/systemd-timesyncd
 1282     1 root     S<   15028   3%   0% python /usr/sbin/phosphor-mapper --path_namespaces=/org/open_power /xyz/openbmc_project /org/openbmc /com/ibm --interface_namespaces=xyz.openbmc_project org.openbmc com.ibm org.open_power --blacklists= --interface_blacklists=
 1365     1 root     S    13136   3%   0% python /usr/sbin/chassis_control.py
 1850     1 root     S    13132   3%   0% python /usr/sbin/system_manager.py
 1870     1 root     S     8552   2%   0% ipmid
 1254     1 root     S     8408   2%   0% phosphor-dbus-monitor
 1798     1 root     S     7796   2%   0% openpower-update-manager
 1857     1 root     S     7776   2%   0% phosphor-image-updater
 1703     1 root     R     7016   2%   0% phosphor-inventory
 1692     1 root     S     6916   2%   0% ibm-log-manager
 1697     1 root     S     6568   2%   0% phosphor-settings-manager
 1233     1 root     S     6540   2%   0% phosphor-hwmon-readd -o /ahb/apb/i2c@1e78a000/i2c-bus@100/bmp280@77
 1029     1 root     S     6532   2%   0% phosphor-user-manager
 1289     1 root     S     6512   2%   0% witherspoon-psu-monitor --path=/sys/bus/i2c/devices/3-0068 --instance=1 --inventory=/system/chassis/motherboard/powersupply1 --num-history-records=120 --sync-gpio-path=/dev/gpiochip0 --sync-gpio-num=105
 1291     1 root     S     6512   2%   0% witherspoon-psu-monitor --path=/sys/bus/i2c/devices/3-0069 --instance=0 --inventory=/system/chassis/motherboard/powersupply0 --num-history-records=120 --sync-gpio-path=/dev/gpiochip0 --sync-gpio-num=105
 1263     1 root     S     6432   1%   0% phosphor-hwmon-readd -o /ahb/apb/i2c@1e78a000/i2c-bus@180/ir35221@71
 1262     1 root     S     6432   1%   0% phosphor-hwmon-readd -o /ahb/apb/i2c@1e78a000/i2c-bus@180/ir35221@70
 1264     1 root     S     6432   1%   0% phosphor-hwmon-readd -o /ahb/apb/i2c@1e78a000/i2c-bus@140/ir35221@71
 1260     1 root     S     6432   1%   0% phosphor-hwmon-readd -o /ahb/apb/i2c@1e78a000/i2c-bus@140/ir35221@70
 1234     1 root     S     6432   1%   0% phosphor-hwmon-readd -o /ahb/apb/i2c@1e78a000/i2c-bus@100/power-supply@68
 1259     1 root     S     6432   1%   0% phosphor-hwmon-readd -o /ahb/apb/i2c@1e78a000/i2c-bus@100/power-supply@69
 1261     1 root     S     6432   1%   0% phosphor-hwmon-readd -o /ahb/apb/i2c@1e78a000/i2c-bus@380/tmp275@4a
 1961     1 root     S     6404   1%   0% phosphor-network-manager
 1855     1 root     S     6396   1%   0% openpower-occ-control
 1026     1 root     S     6304   1%   0% phosphor-log-manager
 1043     1 root     S     6248   1%   0% phosphor-dump-monitor
 1960     1 root     S     6224   1%   0% phosphor-host-state-manager
 1691     1 root     S     6216   1%   0% phosphor-ledmanager
 1851     1 root     S     6200   1%   0% phosphor-network-snmpconf
 1022     1 root     S     6136   1%   0% phosphor-download-manager
 1037     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/cffps1/68
 1211     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/front/power
 1046     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/cffps1/69
 1207     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/power/button
 1208     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/rear/power
 1209     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/rear/id
 1210     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/rear/fault
 1212     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/front/id
 1213     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/front/fault
 1216     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/fan2
 1217     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/fan1
 1800     1 root     S     6056   1%   0% phosphor-bmc-state-manager
 1214     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/fan3
 1215     1 root     S     6056   1%   0% phosphor-ledcontroller -p /sys/class/leds/fan0
 1025     1 root     S     6048   1%   0% phosphor-rsyslog-conf
 1011     1 systemd- S     5900   1%   0% /lib/systemd/systemd-resolved
 1704     1 root     S     5388   1%   0% phosphor-fru-fault-monitor
 1021     1 root     S     5352   1%   0% phosphor-sync-software-manager
  731     1 systemd- S     5328   1%   0% /lib/systemd/systemd-networkd
 1024     1 root     S     5176   1%   0% slpd
 1415  1393 root     S     4868   1%   0% systemctl start xyz.openbmc_project.Hwmon@ahb-apb-i2c\x401e78a000-i2c\x2dbus\x40100-max31785\x4052.service
 1447     1 root     RN    4748   1%   0% /lib/systemd/systemd-coredump
 1425     1 root     DN    4748   1%   0% /lib/systemd/systemd-coredump
 1452     1 root     DN    4748   1%   0% /lib/systemd/systemd-coredump
 1957     1 root     DN    4748   1%   0% /lib/systemd/systemd-coredump
 1958     1 root     DN    4748   1%   0% /lib/systemd/systemd-coredump
 1959     1 root     DN    4748   1%   0% /lib/systemd/systemd-coredump
 1032     1 messageb S     4684   1%   0% /usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only
  716     1 root     S     4060   1%   0% /lib/systemd/systemd-udevd
 1866  1853 root     S     3108   1%   0% {dreport} /bin/bash /usr/bin/dreport -d /var/lib/phosphor-debug-collector/dumps/5 -i 5 -s 200 -q -v -p /var/lib/systemd/coredump/core.phosphor-timema.0.addbf29078794c2aa2e58551a6c25bac.1339.1537179593000000.xz -t core
 1962  1853 root     S     3108   1%   0% {dreport} /bin/bash /usr/bin/dreport -d /var/lib/phosphor-debug-collector/dumps/6 -i 6 -s 200 -q -v -p /var/lib/systemd/coredump/core.phosphor-fru-fa.0.addbf29078794c2aa2e58551a6c25bac.1364.1537179624000000.xz -t core
 2003  1853 root     S     3108   1%   0% {dreport} /bin/bash /usr/bin/dreport -d /var/lib/phosphor-debug-collector/dumps/8 -i 8 -s 200 -q -v -p /var/lib/systemd/coredump/core.openpower-updat.0.addbf29078794c2aa2e58551a6c25bac.1243.1537179588000000.xz -t core
 1993  1853 root     S     3108   1%   0% {dreport} /bin/bash /usr/bin/dreport -d /var/lib/phosphor-debug-collector/dumps/7 -i 7 -s 200 -q -v -p /var/lib/systemd/coredump/core.phosphor-host-s.0.addbf29078794c2aa2e58551a6c25bac.1346.1537179596000000.xz -t core
 1137  1108 root     S     2976   1%   0% -sh
 1393     1 root     S     2976   1%   0% {start_max31785_} /bin/sh /usr/bin/start_max31785_hwmon.sh ahb/apb/i2c@1e78a000/i2c-bus@100/max31785@52
 1709     1 root     S     2944   1%   0% mapper wait /xyz/openbmc_project/state/host0
 2172  1866 root     R     2932   1%   0% [mkdir]
 2174  1993 root     D     2932   1%   0% mkdir -p /var/lib/phosphor-debug-collector/dumps/7
 2175  1962 root     R     2932   1%   0% mkdir -p /var/lib/phosphor-debug-collector/dumps/
 1031     1 root     S     2876   1%   0% btbridged
 1108     1 root     S     2440   1%   0% /usr/sbin/dropbear -i -r /var/lib/dropbear/dropbear_rsa_host_key -B
 1023     1 root     S     1864   0%   0% obmc-console-server --config /etc/obmc-console.conf ttyVUART0
 1110     1 root     S     1864   0%   0% /sbin/agetty -8 -L ttyS4 115200 xterm
 1036     1 root     S     1728   0%   0% /usr/sbin/rngd -f
 1230     2 root     DW       0   0%   0% [kworker/u2:1+fl]
    7     2 root     RW       0   0%   0% [ksoftirqd/0]
  176     2 root     IW       0   0%   0% [kworker/u2:3-ev]
  538     2 root     RW       0   0%   0% [kworker/0:2-eve]
   11     2 root     IW       0   0%   0% [kworker/0:1-eve]
  574     2 root     SW       0   0%   0% [ubifs_bgt0_0]
    5     2 root     IW       0   0%   0% [kworker/u2:0-ev]
  292     2 root     SW       0   0%   0% [hwrng]
  571     2 root     IW<      0   0%   0% [kworker/0:1H-kb]
    8     2 root     SW       0   0%   0% [watchdog/0]
    3     2 root     IW       0   0%   0% [kworker/0:0-eve]
    9     2 root     SW       0   0%   0% [kdevtmpfs]
  209     2 root     SW       0   0%   0% [watchdogd]
 1382     2 root     IW       0   0%   0% [kworker/u2:2-ev]
 2178  2003 root     RW       0   0%   0% [basename]
    2     0 root     SW       0   0%   0% [kthreadd]
    4     2 root     IW<      0   0%   0% [kworker/0:0H-kb]
    6     2 root     IW<      0   0%   0% [mm_percpu_wq]
   87     2 root     SW       0   0%   0% [oom_reaper]
  185     2 root     IW<      0   0%   0% [writeback]
  187     2 root     IW<      0   0%   0% [crypto]
  189     2 root     IW<      0   0%   0% [kblockd]
  236     2 root     SW       0   0%   0% [kswapd0]
  518     2 root     IW<      0   0%   0% [ipv6_addrconf]
  564     2 root     SW       0   0%   0% [ubi_bgt4d]
 1047     2 root     IW       0   0%   0% [kworker/0:3-eve]
 1071     2 root     SW       0   0%   0% [ubi_bgt8d]
 1092     2 root     SW       0   0%   0% [ubifs_bgt8_1]
 1100     2 root     SW       0   0%   0% [ubifs_bgt8_2]
 1182     2 root     SW       0   0%   0% [ubifs_bgt4_0]
 1240     2 root     IW       0   0%   0% [kworker/0:4-eve]
 1457     2 root     SW       0   0%   0% [ubifs_bgt8_3]
```
- [x] Journald log 
[BMC_service_crashes.txt](https://github.com/openbmc/openbmc/files/2388789/BMC_service_crashes.txt)

**Comment by geissonator** [comment]
https://github.com/openbmc/openbmc/issues/3369 is tracking the root issue here (which is still unknown). I'd like to use this bug to track the side affects of this bug. We should not have services endlessly restarting.

Because the timeout on a service default to 90s, and we allow up to 2 restarts before giving up, we need our StartLimitIntervalSec to be relative to this.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/12874> Increase StartLimitIntervalSec to 240s
Resolves  #3379 BMC goes unstable on resets

**Comment by geissonator** [comment]
Had to revert my change due to https://github.com/openbmc/openbmc/issues/3393, so reopening.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/14745> Disable systemd-coredump from phosphor layer
resolves  #3379 BMC goes unstable on resets
